controllers/documento_presupuestal.go

```diff
@@ -191,3 +191,44 @@ func (j *DocumentoPresupuestalController) GetDocMovByParent() {
 	j.Data["json"] = response
 	j.ServeJSON()
 }
+
+// GetDocMovByRubro ...
+// @Title GetDocMovByRubro
+// @Description Obtiene todos los CDPs expedidos con movimientos en un rubro padre dado
+// @Param	vigencia	path	int	true	"Vigencia del CDP"
+// @Param	rubro	path	string	true	"Rubro padre relacionado al CDP"
+// @Param areaFuncional path  int    true  "Area Funcional"
+// @Success 200 {object} []models.DocumentoPresupuestal Listado de documentos relacionados
+// @Failure 500 Internal server error
+// @router /get_doc_mov_rubro/:vigencia/:areaFuncional/:rubro [get]
+func (j *DocumentoPresupuestalController) GetAllDocMovByRubro() {
+
+	vigencia := j.Ctx.Input.Param(":vigencia")
+	rubro := j.Ctx.Input.Param(":rubro")
+	centroGestor := j.Ctx.Input.Param(":areaFuncional")
+
+	docPresComp := compositors.DocumentoPresupuestalCompositor{}
+	docs, err := docPresComp.GetAllDocumentoPresupuestalMovimientosByRubro(vigencia, centroGestor, rubro)
+	j.Data["json"] = DefaultResponse(0, err, &docs)
+	j.ServeJSON()
+}
+
+// GetInfoCrp ...
+// @Title GetInfoCrp
+// @Description Obtiene los RP asociados a un CDP y a un contratista en una vigencia
+// @Param	vigencia	path	string	true	"Vigencia del CRP"
+// @Param	cdp	path	string	true	"Numero consecutivo del CDP asociado"
+// @Param personaId path  string    true  "Numero de documento de la persona"
+// @Success 200 {object} []models.DocumentoPresupuestal Listado de documentos relacionados
+// @Failure 500 Internal server error
+// @router /get_info_crp/:vigencia/:cdp/:personaId [get]
+func (j *DocumentoPresupuestalController) GetInfoCrp() {
+	vigencia := j.Ctx.Input.Param(":vigencia")
+	cdp := j.Ctx.Input.Param(":cdp")
+	personaId := j.Ctx.Input.Param(":personaId")
+
+	docPresComp := compositors.DocumentoPresupuestalCompositor{}
+	docs, err := docPresComp.GetRpByPersonaId(vigencia, cdp, personaId)
+	j.Data["json"] = DefaultResponse(0, err, &docs)
```

Comment:
```suggestion
	j.Data["json"] = DefaultResponse(http.StatusOK, err, &docs)
```
Mejor aún, que el código de estado se defina según si el `err`or fue `nil` o no